tools/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -131,6 +135,7 @@ public partial class OutlineService : ServiceBase {
 
     private void HandleConnection(IAsyncResult result) {
       eventLog.WriteEntry("Got incoming connection");
+      eventLog.WriteEntry(GetNetworkInfo());

```

Comment:
Left this in by mistake, but will wait to change (compile, sign the service) until the review is done 😅